test/rekt/sink_binding_test.go

```diff
@@ -0,0 +1,58 @@
+// +build e2e
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rekt
+
+import (
+	"testing"
+
+	"knative.dev/eventing/test/rekt/features/sinkbinding"
+	"knative.dev/eventing/test/rekt/resources/deployment"
+	sb "knative.dev/eventing/test/rekt/resources/sinkbinding"
+	"knative.dev/eventing/test/rekt/resources/svc"
+	"knative.dev/reconciler-test/pkg/feature"
+)
+
+// TestSmoke_SinkBinding
+func TestSmoke_SinkBinding(t *testing.T) {
+	t.Parallel()
+
+	ctx, env := global.Environment()
+	t.Cleanup(env.Finish)
+
+	names := []string{
```

Comment:
So, the only thing different between these names is the formatting of the name so I guess I'm still trying to see if there's a way to separate the creation of the resource, and if the TestSmoke_SinkBinding only would take t, and coordinates (ns, name) of the resource under test, then moving to a world where the SinkBinding (or any other resource) gets created and only the tests specified will run against that resource.
Maybe I'm thinking about this wrong, but that way there would be zero knowledge necessary (we could still have tests like this, but split into two pieces) between construction vs. testing. So just being able to point to a "closed box" resource under test and observe the results would be what I think we'd want in the long term. I'm just jotting down my thoughts here.